services/apps/alcs/src/alcs/application-decision/application-decision-v2/application-decision/application-decision-v2.controller.ts

```diff
@@ -198,6 +204,55 @@ export class ApplicationDecisionV2Controller {
       modifies,
       reconsiders,
     );
+
+    // Send email after releasing draft
+    if (updateDto.isDraft === false) {
```

Comment:
I think this is only supposed to happen when the first decision is released, and reverting to draft then releasing again shouldn't do that.

Reply:
Put this block into its own function